tests/vector_operators/generate_vector_operators.py

```diff
@@ -1355,7 +1355,7 @@ def generate_all_types_specific_return_type_test(type_str, size):
         type=type_str,
         size=str(size),
         swizzle=get_swizzle(size),
-        ret_type=Data.opencl_sized_return_type_dict[type_str],
+        ret_type='rel_log_ret_t<{}>'.format(type_str),
```

Comment:
Use `f`-string instead:
```suggestion
        ret_type=f'rel_log_ret_t<{type_str}>',
```